pkgs/tools/admin/zbctl/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "zbctl";
+  version = "8.0.6";
+
+  builder = ./builder.sh;
+
+  inherit coreutils;
```

Comment:
Why is this done?

Reply:
on linux inside a nix container I had trouble building because mkdir, cp, etc where not present without importing coreutils

Reply:
No longer neccesary with the buildphases/stdenvNoCC